libs/orders/src/lib/components/order-list/order-list.tsx

```diff
@@ -218,7 +219,14 @@ export const OrderListTable = memo(
                 return `${Schema.OrderTimeInForceMapping[value]}: ${expiry}`;
               }
 
-              return value ? Schema.OrderTimeInForceMapping[value] : '';
+              const tifLabel = value
+                ? Schema.OrderTimeInForceMapping[value]
```

Comment:
chore (blocking): This looks incorrect. This is the status column we should be using the status mapping

Reply:
oh nvm my mistake